.github/workflows/tests.yml

```diff
@@ -0,0 +1,19 @@
+name: Unit tests
+
+on:
+  push:
+    branches: [ dev, master ]
+  pull_request:
+    branches: [ dev, master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Run tests
+      run: bazel test //...
```

Comment:
Pls remind me to reproduce this in the teamcity build server

Reply:
currently it doesn't work, but i will fix it